lldb/source/Target/SwiftLanguageRuntimeDynamicTypeResolution.cpp

```diff
@@ -304,6 +304,15 @@ class LLDBMemoryReader : public swift::remote::MemoryReader {
       *result = m_process.GetAddressByteSize(); // FIXME: sizeof(size_t)
       return true;
     }
+    case DLQ_GetLeastValidPointerValue: {
+      auto *result = (uint64_t *)outBuffer;
+      auto &triple = m_process.GetTarget().GetArchitecture().GetTriple();
+      if (triple.isOSDarwin() && triple.isArch64Bit())
+        *result = 0x100000000;
+      else
+        *result = 0x1000;
+      return true;
```

Comment:
This was previously unimplemented, and so would return false for `DLQ_GetLeastValidPointerValue` queries. This caused `projectEnumValue` in Swift reflection to fail.

Reply:
See also rdar://70729149